kpatch-build/create-diff-object.c

```diff
@@ -2913,8 +2992,8 @@ static void kpatch_create_patches_sections(struct kpatch_elf *kelf,
 		if (sym->bind == STB_LOCAL && symbol.global)
 			ERROR("can't find local symbol '%s' in symbol table", sym->name);
 
-		log_debug("lookup for %s: obj=%s sympos=%lu size=%lu",
-			  sym->name, symbol.objname, symbol.sympos,
+		log_debug("lookup for %s: obj=%s sympos=%lu size=%lu\n",
+		          sym->name, symbol.objname, symbol.sympos,
```

Comment:
nit: adding `\n` is good, but the second line introduces spaces (in front of `sym->name`) instead of a tab